platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/layers/property_factory.java.ejs

```diff
@@ -87,22 +87,28 @@ public class PropertyFactory {
 <% } -%>
   /**
    * Converts Android color int to "rbga(r, g, b, a)" String equivalent.
+   * <p>
+   * Alpha value will be converted from 0-255 range to 0-1.
+   * </p>
    *
    * @param color Android color int
    * @return String rgba color
    */
   public static String colorToRgbaString(@ColorInt int color) {
-    return String.format(Locale.US, "rgba(%d, %d, %d, %d)",
-      (color >> 16) & 0xFF, (color >> 8) & 0xFF, color & 0xFF, (color >> 24) & 0xFF);
+    return String.format(Locale.US, "rgba(%d, %d, %d, %.1f)",
```

Comment:
I don't think precision to only one decimal point will be enough in this case. What about increasing this to 3?